### PR TITLE
Re Order future added

### PR DESCRIPTION
**Pull request by surya-ultiweb**

### CODE REVIEW

File: app/Http/Controllers/PurchaseHistoryController.php [hunk 1]
```diff
@@ -276,4 +279,121 @@ public function returnMail($order, $order_name)
...
+
+			if (!empty($tax_name) && $tax_name->name == 'IGST') {
+				$insertcart['tax1'] =  $splitTax;
+				$insertcart['tax1_amount'] = (($cut_pr));
+
+				$insertcart['tax2'] =  $splitTax;
+				$insertcart['tax2_amount'] = (($cut_pr));
+			}
+		}
+		$insertcart['shipping_cost'] = 0;
```
Comment (priya-ultiweb) [inline app/Http/Controllers/PurchaseHistoryController.php:391]: Why is shipping cost 0 ? Can you pls confirm if we cannot take shipping cost from DB ?

File: resources/views/frontend/user/order_details_customer.blade.php [hunk 2]
```diff
@@ -106,6 +113,7 @@
                                 @endif
                                 <th data-breakpoints="md" class="text-right">{{ translate('Status') }}</th>
                                 <th data-breakpoints="md" class="text-right">{{ translate('Return') }}</th>
+                                <th data-breakpoints="md" class="text-right">{{ translate('Re Order') }}</th>
```
Comment (priya-ultiweb) [inline resources/views/frontend/user/order_details_customer.blade.php:116]: We spoke about having one column for both Return and Re-order - right - Can you pls confirm we can achieve that ?

**Review by priya-ultiweb — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/PurchaseHistoryController.php (hunk 1 above)
Comment (surya-ultiweb) [reply app/Http/Controllers/PurchaseHistoryController.php:391]: Shipping cost will be calculated while checkout the product

File: resources/views/frontend/user/order_details_customer.blade.php (hunk 2 above)
Comment (surya-ultiweb) [reply resources/views/frontend/user/order_details_customer.blade.php:116]: i tried that once i put this in same column other buttons it's not appeared

Comment (surya-ultiweb) [reply resources/views/frontend/user/order_details_customer.blade.php]: > We spoke about having one column for both Return and Re-order - right - Can you pls confirm we can achieve that ?

changes done and pushed.

**Review by priya-ultiweb — APPROVED** [review]

**Review by priya-ultiweb — APPROVED** [review]

**Pull request merged**